job-dsls/jobs/kie/main/deploy_jobs.groovy

```diff
@@ -54,56 +62,35 @@ def final REPO_CONFIGS = [
                 mvnProps               : DEFAULTS["mvnProps"] + [
                         "gwt.compiler.localWorkers": "2"
                 ],
-                downstreamRepos        : ["droolsjbpm-knowledge"]
+                downstreamRepos        : ["kie-uberfire-extensions"]
         ],
-        "droolsjbpm-knowledge"      : [
+        "kie-uberfire-extensions"   : [
                 timeoutMins            : 40,
-                downstreamRepos        : ["drools"]
-        ],
-        "drools"                    : [
-                downstreamRepos        : ["optaplanner-7.x", "jbpm", "kie-jpmml-integration"],
-                artifactsToArchive     : ["**/target/testStatusListener*"]
-        ],
-        "optaplanner"               : [
-                mvnGoals: "-e -fae -B clean deploy com.github.spotbugs:spotbugs-maven-plugin:spotbugs",
-                mvnProps: [
-                        "full"                     : "true",
-                        "integration-tests"        : "true",
-                        "maven.test.failure.ignore": "true"
-                ]
+                downstreamRepos        : ["kie-wb-common"]
         ],
         "jbpm"                      : [
                 timeoutMins            : 120,
                 mvnGoals               : DEFAULTS["mvnGoals"] + " -Dcontainer.profile=wildfly",
-                downstreamRepos        : ["jbpm-work-items", "kie-jpmml-integration"]
+                downstreamRepos        : ["kie-jpmml-integration"]
         ],
         "kie-jpmml-integration"     :[
                 downstreamRepos        : ["droolsjbpm-integration"]
```

Comment:
This is also wrong, there is no downstream repo for kie-jpmml-integration. And droolsjbpm-integration should be in jbpm downstreamRepos too